test/slotted.html

```diff
@@ -0,0 +1,195 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+  <title>iron-form</title>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../web-component-tester/browser.js"></script>
+
+  <link rel="import" href="../../polymer/polymer.html">
+  <link rel="import" href="../../paper-checkbox/paper-checkbox.html">
+  <link rel="import" href="../../paper-input/paper-input.html">
+  <link rel="import" href="../iron-form.html">
+
+</head>
+<body>
+  <dom-module id="slotted-form">
```

Comment:
Mind adding a test for 2 level distribution? e.g.

```html
<dom-module id="slotted-slotted-form">
    <template>
      <slotted-form id="form">
         <input slot="inputs" name="slotted" value="works too" />
         <slot slot="inputs" name="inputs"></slot>
      </slotted-form>
    </template>
    <script>
    HTMLImports.whenReady(function() {
      Polymer({is: 'slotted-slotted-form'});
    });
    </script>
</dom-module>

<slotted-slotted-form>
   <input slot="inputs" name="input" value="some" />
</slotted-slotted-form>

Reply:
Ah good call, it actually uncovered a bug! This might have been the issue @rbjarnason was encountering as well.